Test failure in ydb/tests/compatibility/test_fulltext_index.py.TestFulltextIndex.test_fulltext_index[rolling_stable-26-1-1_to_prestable-26-2]: Member not found: pk

## Summary

During rolling upgrade from `stable-26-1-1` to `prestable-26-2`, a query against a fulltext index fails with `GenericError: "Member not found: pk"`. The `pk` column is not visible in the index result set during the mixed-version state.

Previously similar issue: #33775 (closed, version pair `prestable-26-1_to_prestable-26-2`, same error).

## Affected test (persistent failure, all 3 tries)

- `ydb/tests/compatibility/test_fulltext_index.py.TestFulltextIndex.test_fulltext_index[rolling_stable-26-1-1_to_prestable-26-2]`

## Build Information

- **Commit:** `7a77c4d3eab36d99c9b938656150b779a90e1e10`
- **Build Preset:** relwithdebinfo
- **Branch:** main
- **Date:** 2026-02-26

## Error details

```
ydb/tests/compatibility/test_fulltext_index.py:176: in test_fulltext_index
    self.wait_index_ready()
ydb/tests/compatibility/test_fulltext_index.py:81: in wait_index_ready
    assert wait_for(predicate, timeout_seconds=100, step_seconds=1), "Error getting index status"
ydb/tests/library/common/wait_for.py:16: in wait_for
    if predicate():
ydb/tests/compatibility/test_fulltext_index.py:74: in predicate
    self.select_from_index_without_roll()
ydb/tests/compatibility/test_fulltext_index.py:160: in select_from_index_without_roll
    self._do_queries(queries)
ydb/tests/compatibility/test_fulltext_index.py:146: in _do_queries
    result_sets = session_pool.execute_with_retries(query)
contrib/python/ydb/py3/ydb/query/pool.py:259: in execute_with_retries
    return retry_operation_sync(wrapped_callee, retry_settings)
contrib/python/ydb/py3/ydb/retries.py:163: in retry_operation_sync
    for next_opt in opt_generator:
contrib/python/ydb/py3/ydb/retries.py:118: in retry_operation_impl
    result = YdbRetryOperationFinalResult(callee(*args, **kwargs))
...
contrib/python/ydb/py3/ydb/issues.py:266: in _process_response
    raise exc_class(_format_response(response_proto), response_proto.issues)
E   ydb.issues.GenericError: message: "Type annotation" issue_code: 1030 severity: 1
    issues {
        position { row: 2 column: 17 }
        message: "At function: RemovePrefixMembers"
        issues {
            position { row: 6 column: 17 }
            message: "At function: Take"
            issues {
                position { row: 2 column: 17 }
                message: "At function: PersistableRepr, At function: SqlProject, At tuple, At function: SqlProjectItem, At lambda"
                issues {
                    position { row: 2 column: 24 }
                    message: "At function: Member"
                    issues {
                        position { row: 2 column: 24 }
                        message: "Member not found: pk"
                    }
                }
            }
        }
    } (server_code: 400080)
```

The query that fails is a `SELECT pk, text FROM table_string VIEW idx_fulltext_plain_standard WHERE FulltextMatch(...)`.

## Test Reports

- **Workflow run:** [Regression-run_compatibility #448](https://github.com/ydb-platform/ydb/actions/runs/22429562196)
- **HTML reports:** [Try 1](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/22429562196/ya-main-x86-64/try_1/ya-test.html) | [Try 2](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/22429562196/ya-main-x86-64/try_2/ya-test.html) | [Try 3](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/22429562196/ya-main-x86-64/try_3/ya-test.html)

**Logs:**
- [try 1](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/22429562196/ya-main-x86-64/try_1/artifacts/logs/ydb/tests/compatibility/test-results/py3test/testing_out_stuff/test_fulltext_index.py.TestFulltextIndex.test_fulltext_index.rolling_stable-26-1-1_to_prestable-26-2.log)
- [Artifacts Index](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/22429562196/ya-main-x86-64/index.html)

## Diagnosis

During rolling upgrade from `stable-26-1-1` to `prestable-26-2`, a query against a fulltext index fails with "Member not found: pk". This is likely a schema change or index metadata incompatibility between versions where the `pk` column is not visible in the index result set during the mixed-version state.

## Related

- Previously similar (closed): #33775 (same error, version pair `prestable-26-1_to_prestable-26-2`)
- Mute (closed): #34115
- Currently muted in: #35013

--- a/ydb/tests/compatibility/test_fulltext_index.py
+++ b/ydb/tests/compatibility/test_fulltext_index.py
@@ -93,10 +93,10 @@ class TestFulltextIndex(RollingUpgradeAndDowngradeFixture):
         index_name = f"idx_{index_type}_{tokenizer}"
         queries = []
         for i in range(self.query_count):
-            query = ' '.join(self.good_queries[table_name][i])
+            query = self.good_queries[table_name][i]
             queries.append([
                 True, f"""
-                SELECT `pk`, `text`
+                SELECT `key`, `text`
                 FROM `{table_name}`
                 VIEW `{index_name}`
                 WHERE FulltextMatch(`text`, "{query}")
@@ -106,10 +106,11 @@ class TestFulltextIndex(RollingUpgradeAndDowngradeFixture):
             if index_type == 'fulltext_relevance':
                 queries.append([
                     True, f"""
-                    SELECT `pk`, `text`, FulltextScore(`text`, "{query}") as `rel`
+                    SELECT `key`, `text`, FulltextScore(`text`, "{query}") as `rel`
                     FROM `{table_name}`
                     VIEW `{index_name}`
-                    ORDER BY `rel`
+                    WHERE FulltextScore(`text`, "{query}") > 0
+                    ORDER BY `rel` DESC
                     LIMIT {self.limit};
                     """
                 ])

